VideoWeb/.vscode/settings.json

```diff
@@ -3,5 +3,5 @@
   "editor.rulers": [120, 140],
   "dotnet-test-explorer.testProjectPath": "**/*[Tt]ests/*.csproj",
   "markiscodecoverage.searchCriteria": "**/lcov*.info",
-  "prettier.configPath": ".prettierrc"
```

Comment:
I don't think .vscode shouldbe included in the repo. It should be in the .gitignore

Reply:
I would agree but it's in the repo and it's wrong so I figured I should fix it